### PR TITLE
Put reader in 'node' dir

### PR DESCRIPTION
**Pull request by duglin**
W/o this change the reader might not realize they need to cd into the
'node' dir to see the app.js file that we talk about next.

Also removed some extra whitespace at the end of some lines.

Wasn't sure which branch to base this off of, let me know if you want it moved.

Signed-off-by: Doug Davis <dug@microsoft.com>

### CODE REVIEW

File: hello-world/README.md [hunk 1]
```diff
@@ -11,27 +11,27 @@ Later on, you'll deploy a Python app to act as the publisher. The architecture d
...
 
-## Step 1 - Setup Dapr 
+## Step 1 - Setup Dapr
 
 Follow [instructions](https://docs.dapr.io/getting-started/install-dapr/) to download and install the Dapr CLI and initialize Dapr.
 
 ## Step 2 - Understand the code
 
-Now that Dapr is set up locally, clone the repo, then navigate to the Hello World quickstart: 
+Now that Dapr is set up locally, clone the repo, then navigate to the node version of the Hello World quickstart:
```
Comment (paulyuk) [inline hello-world/README.md:24]:
```suggestion
Now that Dapr is set up locally, clone the repo, then navigate to the Node.js version of the Hello World quickstart:
```

**Review by paulyuk — CHANGES_REQUESTED** [review]
Thanks Doug for the contributions and nice catch - Step 2 did not sync with the ./node changes in Step 3.  One small thing to tweak for consistency, and then LGTM.

File: hello-world/README.md (hunk 1 above)
Comment (paulyuk) [reply hello-world/README.md:24]: Oh for @greenie-msft, should this get pushed to the feature_newquickstarts branch, /tutorials/.. folder too?

**Review by paulyuk — DISMISSED** [review]
Lgtm

**Comment by duglin** [comment]
rebased - should be good to go now.

**Review by paulyuk — APPROVED** [review]

**Pull request merged**